code/__DEFINES/is_helpers.dm

```diff
@@ -198,6 +198,8 @@ GLOBAL_LIST_INIT(turfs_openspace, typecacheof(list(
 
 #define isfood(A) (istype(A, /obj/item/reagent_containers/food))
 
+#define isweapon(A) (istype(A, /obj/item/weapon))
+
```

Comment:
Also we should rather be getting rid of item/weapon than giiving it new helpers

Reply:
Well I don't remember why I added this, but I apparently ended up not even using it so in the bin it goes.